opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -25,6 +25,8 @@
 
 # target library to desired instrumentor path/versioned package name
 instrumentations = {
+    "aiohttp-client": "opentelemetry-instrumentation-aiohttp-client>=0.15b0",
```

Comment:
could you add the comment I asked above this, maybe right below the imports?

Reply:
Sweet that is super helpful! Added a comment :)